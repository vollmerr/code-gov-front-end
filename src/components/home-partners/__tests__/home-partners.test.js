import React from 'react';

import HomePartners from 'components/home-partners'

import { shallow } from 'enzyme';

describe('Testing <HomePartners /> component', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<HomePartners />);
  });
  it('Testing if component renders', () => {
    expect(wrapper.length).toEqual(1);
  });
  it('should render correct', () => {
    expect(wrapper).toMatchSnapshot();
  })
})