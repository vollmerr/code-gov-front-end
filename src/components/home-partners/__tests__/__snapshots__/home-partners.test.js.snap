// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <HomePartners /> component should render correct 1`] = `
<div
  className="block featured-project"
  id="partners"
>
  <div
    className="indented"
  >
    <hr />
    <h2>
      Partners
    </h2>
    <p>
      Code California is an open collaboration between California state government agencies, industry partners and civic technologists. Participating agencies:
    </p>
    <div
      className="links"
    >
      <span>
        <a
          href="https://www.govops.ca.gov/"
        >
          <img
            alt="California GovOps logo"
            src="../../../assets/img/logos/govops.png"
          />
        </a>
      </span>
      <span>
        <a
          href="https://cdt.ca.gov/"
        >
          <img
            alt="California Department of Technology logo"
            src="../../../assets/img/logos/cdt.png"
          />
        </a>
      </span>
    </div>
  </div>
</div>
`;
